docs/content/querying/post-aggregations.md

```diff
@@ -54,6 +54,21 @@ The constant post-aggregator always returns the specified value.
 { "type"  : "constant", "name"  : <output_name>, "value" : <numerical_value> }
 ```
 
+### Greatest / Least post-aggregators
```

Comment:
Like I mentioned I choose the name `Greatest` and `Least` like the [MySQL](http://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#function_greatest) and [PostgreSQL](https://www.postgresql.org/docs/8.4/static/functions-conditional.html#AEN15322) functions. SQL also has MAX and GREATEST which are almost the same but different. This also opens up the option to introduce these greatest/least functions as normal aggregators in the future.